app/models/invite_code.rb

```diff
@@ -0,0 +1,23 @@
+require 'coupon_code'
+
+class InviteCode < CouchRest::Model::Base
+  use_database 'invite_codes'
+  property :invite_code, String, :read_only => true
+  property :invite_count, Integer, :default => 0, :accessible => true
+
+  timestamps!
+
+  design do
+    view :by__id
```

Comment:
Yes. Please remove it. It's a quick change and it takes much longer to figure out why this is there later on.